### DISCUSSION TITLE
월드내에 특정위치에 가면 스타트 되게 하려면 어떤 스크립트를 참고해야하나요?

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#940] [4. General]
안녕하세요, 
빌드잇에서 공간구성을 마친후 , 유니티로 가지고와 특정 장소 (예를들어 문)에 캐릭터가 가까이 다가가면 '스타트'라는 버튼이 떠서, 버튼을 클릭했을시 공 피하기 월드가 생성되도록 (공에 맞았을 시 다시 문으로 캐릭터가 되돌아옴) 을 만드려고 하는데,, 어떤 스크립트를 참고하면 되나요...? ㅠ

**Comment by YSHFM** [comment]
OnTriggerEnter OnTriggerStay등을 참고하시면 될 것 같습니다.
https://docs.unity3d.com/kr/530/ScriptReference/MonoBehaviour.OnTriggerEnter.html
트리거 발동하면 멤버변수로 갖고 있는 ui를 SetActive(true)를 하거나 하는 방식이면 되겠지요.
